pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -719,7 +662,7 @@ workflow {
 
     // Prepare busco protein set:
     prepareBusco(params.busco_proteins)
-
+    
```

Comment:
Come on, you know it's the right thing to do. 🙂
```suggestion

```

Reply:
Okay! :)